### ISSUE TITLE
[Build Tools] Gulp is not reporting theme.less errors when not on lines 5 or 46

### ISSUE THREAD

**Issue by bundyo** [Semantic-Org/Semantic-UI#7004]
### Steps
Can happen if one of the used variables in theme.less is overridden somewhere else and the path can't be found.

### Expected Result
To report such errors.

### Actual Result
Build completes silently successful, but doesn't build from the error on.

### Version
2.5.0 (next branch), but probably happens on previous versions too.

**Comment by lubber-de** [comment]
Already fixed in the community fork [Fomantic-UI](https://fomantic-ui.com) since v2.7.8

**Comment by bundyo** [comment]
I don't see how that helps this bug here.

**Comment by caseydlvr** [comment]
> I don't see how that helps this bug here.

It doesn't, but Semantic-UI hasn't had an update on master for around 2 years at this point. Fomantic is a fork that is intended to be merged back into Semantic once (if) the owner of Semantic returns or allows others to approve pull requests. So if you are looking to make contributions that will actually see a release, Fomantic is currently the place to do it.

**Comment by bundyo** [comment]
That is the reason for the PR to not be directed at master.

The bug was reported together with the PR, when I was about to use Semantic UI from its next branch, because Fomantic UI doesn't have an Ember integration and not a clear path (or any mention) to use the old Semantic UI one. Should I delete it? 

Since then, I did transition to Fomantic UI and did several workarounds to be able to use it with the Semantic UI Ember integration and have a separate compilation from the project styles. This brought down the project styles generation from 10 to under 1 second on every change. I did notice a rather severe bug in Fomantic too (in the dropdown module), but hadn't have the time to isolate it and report it.

From what I've seen, Fomantic UI is past the merging back point with its 3.0 plans, but maybe I'm not seeing everything.

**Comment by caseydlvr** [comment]
AFAIK their hope is still to merge back w/ 3.0, but it has admittedly been hard to follow since their plans for what 3.0 will be have changed multiple times. I think it's more of a question of whether SUI would event accept 3.0 if it's a complete architectural rewrite though. It sounds like development on 3.0 has slowed anyway, so who knows if and when it will be completed. Meanwhile FUI is still chugging away at 2 as well. 

I think the intention of the original response by lubber was more to call to attention the existence of FUI and the the fact that it has been under active development while SUI has been dormant (I've run into a few SUI users that did not know this), but it turns out you are already aware of that.
